gedditlive/GedditSDK
# Release 1.0.14

Previously bitcode was not enable.

# Changelog

```
### Changed:

- Events are sent to Geddit server only

### Added:

- Track session event using show id
```

# Testing

In order to test using this branch, you should edit the cocoapods repository used as a source for your Podfile:

1. Make sure that `~/.cocoapods/repos/gedditlive` exists
    - If not, Run `pod install` with the following Podfile:

```ruby
source 'https://github.com/gedditlive/GedditSDK.git'
target 'YOUR_TARGET' do
  use_frameworks!

  pod 'GedditCoreSDK', '1.0.14'
  pod 'GedditCommerceSDK', '1.0.14'
end
```

2. Switch to the `release-1.0.14` branch:

```sh
cd ~/.cocoapods/repos/gedditlive;
git checkout release-1.0.14
```

3. Edit the podspec files to make `pod install` downloads the frameworks on `release-1.0.14` branch.

_~/.cocoapods/repos/gedditlive/GedditCommerceSDK/1.0.14/GedditCommerceSDK.podspec:_
```ruby
s.source = { :http => 'https://github.com/gedditlive/GedditSDK/blob/release-1.0.14/GedditCommerceSDK/1.0.14/GedditCommerceSDK.zip?raw=true'}
```

_~/.cocoapods/repos/gedditlive/GedditCommerceSDK/1.0.14/GedditCoreSDK.podspec:_
```ruby
s.source = { :http => 'https://github.com/gedditlive/GedditSDK/blob/release-1.0.14/GedditCoreSDK/1.0.14/GedditCoreSDK.zip?raw=true'}
```

4. Make sure to clear your pod cache:

```sh
rm -rf ~/Library/Caches/CocoaPods/Pods/Release/GedditCommerceSDK
rm -rf ~/Library/Caches/CocoaPods/Pods/Release/GedditCoreSDK
```

5. Run `pod install` in your project
